includes/helpers_functions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+function format_occurrence($occur) {
+    $occur = str_replace('_', ' ', $occur);
+    return ucwords($occur);
+}
+
+function format_date($date, $format = 'M/d/Y h:i a') {
+    return date($format, strtotime($date));
+}
+
+function get_subscription_plan($plan_id) {
+    global $wpdb;
+    $table_name = $wpdb->prefix . 'banana_crystal_subscription_plans';
+    $result = $wpdb->get_row("SELECT * FROM $table_name  WHERE deleted_at IS NULL AND subscription_plan_id=".$plan_id);
+    return $result;
+}
+
+function get_expiry_date_by_occurence($occur) {
+    $days = 0;
+    switch($occur) {
+        case 'monthly': 
+            $days = 30;
+            break;
```

Comment:
For determining the expiry date , this function will set it at exactly 30/90/180 days afterwards. 

For months that have 28 days and 31 days, there will be an offset of days and hence over time the subscription will be charged on different days of the month...any thoughts?

For example, I subscribe on the 4th of February, then I would expect the next charge to be March 4th,  however 30 days later is March 7th